core/src/main/java/com/linecorp/armeria/client/retry/RetryLimitedException.java

```diff
@@ -0,0 +1,41 @@
+/*
+ * Copyright 2025 LY Corporation
+ *
+ * LY Corporation licenses this file to you under the Apache License,
+ * version 2.0 (the "License"); you may not use this file except in compliance
+ * with the License. You may obtain a copy of the License at:
+ *
+ *   https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ */
+
+package com.linecorp.armeria.client.retry;
+
+/**
+ * An exception thrown when a retry is limited by a {@link RetryLimiter}.
+ */
+public final class RetryLimitedException extends RuntimeException {
+
+    private static final long serialVersionUID = 7203512016805562689L;
+
+    private static final RetryLimitedException INSTANCE = new RetryLimitedException();
+
+    /**
+     * Returns an instance of {@link RetryLimitedException}.
+     */
+    public static RetryLimitedException of() {
```

Comment:
Shouldn't we use the `verboseExceptionSampler`? e.g. https://github.com/line/armeria/blob/e94aceef3266283075fcb5da680f9e29e31f49b7/core/src/main/java/com/linecorp/armeria/client/RefusedStreamException.java#L39-L41